integration-tests/multi_sig/multi_sig.spec.ts

```diff
@@ -0,0 +1,489 @@
+        ///    then the raw tx will be signed by both customer and merchant to make the agreement.
+        let rawTxId: string;
+        try {
+            rawTxId = await createRawTxForMultiSig(
+                customerTxId,
+                merchant,
+                CUSTOMER_PAYMENT, // merchat receive payment
+                customer,
+                CUSTOMER_DEPOSIT, // customer get back deposit
+                [
+                    customer.viewKeyPair.publicKey!,
+                    merchant.viewKeyPair.publicKey!,
+                    escrow.viewKeyPair.publicKey!,
+                ],
+            );
+        } catch (error) {
+            console.error('ERR createRawTxForMultiSig:', error);
+        }
+        console.log('[LOG] construct rawTx successfully:', rawTxId);
+
+        /// 4. co-sign the tx together between customer and merchant
+        /// 4.1 create customer and merchant multiSig session
+        ///     noticed that session state keeps updated in the Builder instance
+        customerMultiSigBuilder.createNewSession(rawTxId);
+        merchantMultiSigBuilder.createNewSession(rawTxId);
+
+        /// 4.2 generate nonce commitment
+        let customerNC = customerMultiSigBuilder.generateNonceCommitment();
+        let merchantNC = merchantMultiSigBuilder.generateNonceCommitment();
+
+        /// 4.3 add nonce commitment
+        customerMultiSigBuilder.addNonceCommitment(
+            merchant.transferKeyPair.publicKey!,
+            merchantNC,
+        );
+        merchantMultiSigBuilder.addNonceCommitment(
+            customer.transferKeyPair.publicKey,
+            customerNC,
+        );
+
+        /// 4.4 generate nonce
+        let customerNonce = customerMultiSigBuilder.generateNonce();
+        let merchantNonce = merchantMultiSigBuilder.generateNonce();
+
+        /// 4.5 add nonce
+        customerMultiSigBuilder.addNonce(
+            merchant.transferKeyPair.publicKey!,
+            merchantNonce,
+        );
+        merchantMultiSigBuilder.addNonce(
+            customer.transferKeyPair.publicKey,
+            customerNonce,
+        );
+
+        /// 4.6 partial sign
+        let customerPS = customerMultiSigBuilder.partialSign();
+        let merchantPS = merchantMultiSigBuilder.partialSign();
+
+        /// 4.7 add partial signature
+        customerMultiSigBuilder.addPartialSignature(
+            merchant.transferKeyPair.publicKey!,
+            merchantPS,
+        );
+        merchantMultiSigBuilder.addPartialSignature(
+            customer.transferKeyPair.publicKey,
+            customerPS,
+        );
+
+        const customerSig = customerMultiSigBuilder.sign();
+        const merchantSig = merchantMultiSigBuilder.sign();
+
+        // two final signature should be the same
+        expect(customerSig).to.equal(merchantSig);
+
+        // broadcast the signed tx
+        try {
+            await tendermintRpc.broadcastTx(customerSig!.toString('hex'));
```

Comment:
A quick question here, @calvinlauco the final output after complete the multiSig flow should be the signature signed by both parties. So at the last step, should I broadcast the [rawTx](https://github.com/crypto-com/chain-nodelib/pull/19/files#diff-30b8bb3152a399895c3309b2734ed6f6R95) or the broadcast the signature?